components/results/ResultsSuccess.tsx

```diff
@@ -0,0 +1,11 @@
+import CenteredCardContent from '../common/CenteredCardContent';
+
+export default function ResultsSuccess(): JSX.Element {
+  return (
+    <CenteredCardContent
+      title="Verification validated"
+      description="You can now close this window. The NEAR Foundation will contact you for the next steps."
```

Comment:
This feels anti-climatic. We just asked someone to go through a bunch of hassle, and the result came back positive, so I think we should say something happier and more positive, especially since we're then bringing them the bad news that they have to wait longer on something else.

Maybe we want to say:

> Congratulations! 🎉 We have verified your identity and are excited to move forward. NEAR Foundation will contact you for the next steps shortly. You can now close this window.

Reply:
Good one! Thanks a lot!